src/components/MenuLink/index.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { MenuLink as ReachMenuLink } from '@reach/menu-button';
+
+const noop = () => {};
+
+const MenuLink = ({ onSelect, children, isDisabled, href, as }) => (
```

Comment:
should `as` have a default value in our wrapper?

Reply:
as is only really needed to use `Link`. Leaving it undefined, reach will render it as an anchor, so I didn't find it necessary.